### PR TITLE
Set print usage request max size in code

### PR DESCRIPTION
**Pull request by kenoir**
This change sets the max request size for print usage only.

### CODE REVIEW

**Comment by akash1810** [comment]
what's the advantage of this over the config file?

**Comment by kenoir** [comment]
@akash1810 it sets the max request size for this route only, also i'm not convinced the config was taking effect.

File: usage/app/lib/Config.scala [hunk 1]
```diff
@@ -25,6 +25,10 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
 
   val defaultPageSize = 100
   val defaultMaxRetries = 6
+  val defaultMaxPrintRequestSizeInKb = 500
```
Comment (akash1810) [inline usage/app/lib/Config.scala]: minor: why not just do `val defaultMaxPrintRequestSizeInKb = 500 * 1024` here and get rid of `maxPrintRequestLength` above as `maxPrintRequestLength` feels like a config constant.

**Comment by akash1810** [comment]
:+1:

File: usage/app/lib/Config.scala (hunk 1 above)
Comment (kenoir) [inline usage/app/lib/Config.scala]: Feels weird to be doing anything except setting config here.

**Pull request merged**